os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -430,16 +403,61 @@ void i2s_set_param(i2s_t *obj, int channel_num, int rate, int word_len)
 		clock_mode = PLL_CLOCK_98P304M / Clock_Params.PLL_DIV;
 		break;
 	}
+	return clock_mode;
+}
+
+/**
+  * @brief  Set I2S channel number, sample rate and word length.
+  * @param  obj: I2S object defined in application software.
+  * @param  channel_num: This parameter can be one of the following values:
+  *   @arg CH_STEREO: Stereo channel.
+  *   @arg CH_MONO: Mono channel.
+  * @param  rate: This parameter can be one of the following values:
+  *   @arg SR_8KHZ: Sample rate is 8kHz.
+  *   @arg SR_12KHZ: Sample rate is 12kHz.
+  *   @arg SR_16KHZ: Sample rate is 16kHz.
+  *   @arg SR_24KHZ: Sample rate is 24kHz.
+  *   @arg SR_32KHZ: Sample rate is 32kHz.
+  *   @arg SR_48KHZ: Sample rate is 48kHz.
+  *   @arg SR_64KHZ: Sample rate is 64kHz.
+  *   @arg SR_96KHZ: Sample rate is 96kHz.
+  *   @arg SR_192KHZ: Sample rate is 192kHz.
+  *   @arg SR_384KHZ: Sample rate is 384kHz.
+  *   @arg SR_11p025KHZ: Sample rate is 11.025kHz.
+  *   @arg SR_22p05KHZ: Sample rate is 22.05kHz.
+  *   @arg SR_44p1KHZ: Sample rate is 44.1kHz.
+  *   @arg SR_88p2KHZ: Sample rate is 88.2kHz.
+  *   @arg SR_176p4KHZ: Sample rate is 176.4kHz.
+  * @param  word_len: This parameter can be one of the following values:
+  *   @arg WL_16b: Sample bit is 16 bit.
+  *   @arg WL_24b: Sample bit is 24 bit.
+  *   @arg WL_32b: Sample bit is 32 bit.
+  * @retval none
+  */
+void i2s_set_param(i2s_t *obj, int channel_num, int rate, int word_len)
+{
+	uint32_t clock_mode = 0;
+
+	obj->channel_num = channel_num;
+	obj->sampling_rate = rate;
+	obj->word_length = word_len;
+	clock_mode = i2s_clock_select(obj);
+
+	/* Sport Deinit */
+    AUDIO_SP_Unregister(obj->i2s_idx, obj->direction);
```

Comment:
Please check alignment

Reply:
Thank you @edwakuwaku, this is fixed now